package.json

```diff
@@ -8,31 +8,16 @@
   "author": "All Contributors (https://allcontributors.org)",
   "license": "MIT",
   "scripts": {
-    "semantic-release": "semantic-release",
-    "start": "cd website && ../node_modules/.bin/docusaurus-start",
-    "build": "cd website && ../node_modules/.bin/docusaurus-build && cp _redirects build/all-contributors",
-    "write-translations": "cd website && ../node_modules/.bin/docusaurus-write-translations",
-    "crowdin": "java -jar website/crowdin/crowdin-cli.jar",
-    "crowdin-upload": "npm run crowdin -- --config crowdin.yaml upload sources --auto-update -b $(git rev-parse --abbrev-ref HEAD)",
-    "crowdin-download": "npm run crowdin -- --config crowdin.yaml download -b $(git rev-parse --abbrev-ref HEAD)",
-    "build-with-translations": "npm run write-translations && npm run crowdin-upload && npm run crowdin-download && npm run build",
-    "commit": "git-cz"
```

Comment:
@lwasser @flpm Maybe we want to keep this as it helps with the commit convention, or are we abandoning that?

Reply:
I do really like the commit convention tool - I should have waited for your review @JimMadge. The good news is that we are merging into a dev branch here - not main ✨ so we can easily update things. But this was my mistake in merging too quickly.